lib/core/src/lib/form/components/widgets/json/json.widget.scss

```diff
@@ -0,0 +1,11 @@
+.adf {
+    &-json-widget {
+        width: 100%;
+
+        &-label {
+            .adf-label {
+                -webkit-font-smoothing: antialiased;
```

Comment:
I think we have postcss already for all kinds of scenarios, you don't need to target some webkit styles unless it's a bugfix

Reply:
I was not getting the same label style as in other widgets and added that rule